src/rdkafka_conf.c

```diff
@@ -471,6 +471,16 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	  _RK(ssl.crl_location),
 	  "Path to CRL for verifying broker's certificate validity."
 	},
+	/* Andrea Minuto (PKCS12) */
```

Comment:
Please remove your name from the comment, the git history already contains that information and it is not valuable in the source itself.